kyaml/fn/runtime/runtimeutil/runtimeutil.go

```diff
@@ -44,6 +46,8 @@ type FunctionFilter struct {
 
 	// exit saves the error returned from Run
 	exit error
+
```

Comment:
it's private, but a comment would be nice for onestop shopping (avoid hunting for the g/setter)